code/modules/research/designs/machine_designs.dm

```diff
@@ -473,23 +473,23 @@
 	id = "tesla_coil"
 	build_path = /obj/item/circuitboard/machine/tesla_coil
 	category = list ("Misc. Machinery")
-	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING | DEPARTMENTAL_FLAG_SCIENCE
+	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING
 
 /datum/design/board/rad_collector
 	name = "Machine Design (Radiation Collector Board)"
 	desc = "The circuit board for a radiation collector array."
 	id = "rad_collector"
 	build_path = /obj/item/circuitboard/machine/rad_collector
 	category = list ("Misc. Machinery")
-	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING | DEPARTMENTAL_FLAG_SCIENCE
```

Comment:
I think this is because of the pointgen (lol)

Reply:
Never even thought about it since nobody uses those for the hot 10 points per hour that you can reach (except for some single guy on Citadel apparently)

Reply:
should be buffed tbh

Reply:
i tried but the single citadel maintainer who can also make lots of points within a 2h shift said no

Reply:
buff then softcap then :^)

Reply:
cit isnt a downstream anymore so just do it anyway lol